spring-data-cassandra/src/main/java/org/springframework/data/cassandra/config/CqlSessionFactoryBean.java

```diff
@@ -55,6 +56,7 @@
 
 import com.datastax.oss.driver.api.core.CqlSession;
 import com.datastax.oss.driver.api.core.CqlSessionBuilder;
+import com.datastax.oss.driver.shaded.guava.common.base.Splitter;
```

Comment:
Let's stick with core Java or Spring API (e.g. `org.springframework.util.StringUtils.split(…)`).

Reply:
The `StringUtils.split` was my first choice, but this method is returning only two elements: before and after the delimiter. So when using `StringUtils.split`, we won't be able to detect such a situation:
https://github.com/spring-projects/spring-data-cassandra/pull/178/files#diff-dbb3a2d9e16f999f5196b32b5e9820faR90

Reply:
We can also default to `String.split(',')` which should do the same job. Since we don't expect a comma in the hostname or an IPv6 address, we should be safe unless I'm missing a subtlety.